### ISSUE TITLE
Create a download() method for downloading market data

### ISSUE THREAD

**Issue by chiraganand** [xKDR/TSx.jl#100]
MarketData.jl provides the function for downloading market data from yahoo and other sources but the primary method `MarketData.yahoo()` accepts only a single ticker (issue JuliaQuant/MarketData.jl#72). For the download function in TSx, I would want to have something like:
```julia
startdate = DateTime(2012, 10, 1)
options = DownloadOptions(src = "yahoo", period1 = startdate, join = JoinBoth)
download(symbols::AbstractVector{String}, options; on="timestamps")::TS
```

This should join all the tickers together using `JoinBoth` strategy on `timestamps` column and return a single `TS` object.

**Comment by chiraganand** [comment]
Join of multiple objects need to be allowed (issue #40) before download can work properly.

**Comment by ayushpatnaikgit** [comment]
I don't think this feature should be a part of TimeFrames. We need to find hard and push it to `MarketData.jl`

**Comment by ayushpatnaikgit** [comment]
@ajaynshah I think we had decided not to put this function in the package. We need to do a PR to `MarketData.jl`. Shall we close this issue?
